ios/browser/api/brave_shields/adblock_engine.h

```diff
@@ -73,7 +73,7 @@ OBJC_EXPORT
     NS_SWIFT_NAME(addResource(key:contentType:data:));
 
 /// Adds a list of `Resource`s from JSON format
```

Comment:
Could you rename this comment to something like "Use a list of `Resource`s from JSON format"

Reply:
good catch, updated!